cmd/lite/main.go

```diff
@@ -57,6 +58,7 @@ func main() {
 		&ingesterConfig, &configStoreConfig, &rulerConfig, &storageConfig, &schemaConfig, &logLevel)
 	flag.BoolVar(&unauthenticated, "unauthenticated", false, "Set to true to disable multitenancy.")
 	flag.Parse()
+	fmt.Println("VERSION", rulerConfig.RuleFormatVersion)
```

Comment:
Why is this here? Could we have something more descriptive than "VERSION"?

Reply:
Whoops, this was just local dev debugging output. Removed.